src/components/defaultStyleDefinitions.ts

```diff
@@ -348,4 +348,95 @@ export const TagCard: ComponentStyleDefinition ={
             minHeight: "2em"
         }
     }
+}
+
+export const Combobox: ComponentStyleDefinition = {
+    displayName: "Combobox",
+    plugins: ["background", "typography", "border"],
+    defaults: {
+        border: {
+            bottom: {
+                width: "1",
+                style: "solid",
+                colorKey: "colors/borderColor"
+            },
+            top: {
+                width: "1",
+                style: "solid",
+                colorKey: "colors/borderColor"
+            },
+            left: {
+                width: "1",
+                style: "solid",
+                colorKey: "colors/borderColor"
+            },
+            right: {
+                width: "1",
+                style: "solid",
+                colorKey: "colors/borderColor"
+            }
+        },
```

Comment:
Why not just
```suggestion
        border: {
            width: "1",
            style: "solid",
            colorKey: "colors/borderColor"
        },
```

or

```suggestion
        borderWidth: "1",
        borderStyle:  "solid",
        borderColorKey: "colors/borderColor",
```


I think it should translate to something like
```scss
border-width: "1px";
border-style: solid;
border-color: $colors.borderColor;
```

Reply:
Unfortunately, it is not possible.

When changing the border for a widget, users have to specify separate values for all the borders:
![image](https://user-images.githubusercontent.com/92857141/169786324-ff51d9c6-19c0-4310-ab27-5939f2d575ba.png)

So the CSS class has to have different values for all of them, like:
```
border-top: 1px solid #505050;
border-left: 1px solid #505050;
border-bottom: 1px solid #505050;
border-right: 1px solid #505050;
```

to allow users to modify only the border they want (in case they want, for example, to have a different value for the bottom border only)